### PR TITLE
10 implement drag n drop feature

### PR DESCRIPTION
**Pull request by archie1802**

### CODE REVIEW

File: src/frontend/.gitignore
Comment (ARES-79) [inline src/frontend/.gitignore:1]: Is there a reason you added those to json files into the gitignore -- I don't know much but I think we kept them in the React tutorial and I think they could be important for keeping us on the same page code wise

Comment (archie1802) [reply src/frontend/.gitignore:1]: yea your right they shoudlnt be

Comment (archie1802) [reply src/frontend/.gitignore:1]: shoudl be fixed now

File: src/frontend/src/Componenets/TempClass.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./TempClass.css"
+import { Draggable} from '@hello-pangea/dnd';
+
+
+type Props = {
+  content: string;
+  index : number;
+};
+function TempClass ({ content, index} : Props) : JSX.Element {
```
Comment (ARES-79) [inline src/frontend/src/Componenets/TempClass.tsx:10]: Is there a standard way for defining the names and types of props? Should we have our own standard?
I did `props: {id : bigint,  name : string}` in my file.

**Review by ARES-79 — APPROVED** [review]
Thanks for changing the .json thing. Everything looks good.

**Pull request merged**